### ISSUE TITLE
请问方向分类器头部网络clshead 是什么样的结构，在线等

### ISSUE THREAD

**Issue by 1998wy123** [PaddlePaddle/PaddleOCR#7127]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

**Comment by WenmuZhou** [comment]
一个fc

**Comment by 1998wy123** [comment]
def forward(self, x, targets=None):
        x = self.pool(x)
        x = paddle.reshape(x, shape=[x.shape[0], x.shape[1]])
        x = self.fc(x)
        if not self.training:
            x = F.softmax(x, axis=1)
        return x
我看代码里是这么写的，应该是池化、fc、softmax这个流程呀

**Comment by 1998wy123** [comment]
是从cls_head.py那个文件看的

**Comment by WenmuZhou** [comment]
池化是改变shape的，softmax是算分数的，有参数的只有那个fc
